_static/css/custom_1626336655663.css

```diff
@@ -238,19 +232,202 @@ body {
     overflow-x: initial !important;
 }
 
-redoc .redoc-json code {
+#redoc-container .redoc-json code {
     border: none !important;
     background: none !important;
 }
 
-redoc .redoc-json .token.punctuation {
+#redoc-container .redoc-json .token.punctuation {
     color: white !important;
 }
 
-redoc .redoc-json .collapser {
+#redoc-container .redoc-json .collapser {
     color: white !important;
 }
 
+#redoc-container .api-info h1, #redoc-container [data-section-id^="tag"] h1 {
+    font-size: 22px;
+    font-weight: 800;
+    color: #0C0F12 !important;
+}
+
+#redoc-container [data-section-id^="tag"] h2, #redoc-container [data-section-id^="tag"] h4 {
+    font-size: 20px;
+    font-weight: 800;
+    color: #0C0F12 !important;
+    margin-bottom: 8px;
+}
+
+#redoc-container [data-section-id^="tag"] p {
+    margin-bottom: 16px;
+}
+
+#redoc-container [data-section-id^="tag"] .sc-jlyJG.evvbxn p {
+    margin-bottom: 0;
+}
+
+#redoc-container [data-section-id^="tag"] {
+    padding: 0 !important;
+}
+
+#redoc-container .dluJDj {
+    padding: 0 0 20px 0;
+}
+
+#redoc-container .cjtbAK {
+    padding: 0 20px 0 0;
+}
+
+#redoc-container [data-section-id*="/paths/"] + [data-section-id*="/paths/"] {
+    padding-top: 20px !important;
+}
+
+#redoc-container [data-section-id*="/paths/"] + [data-section-id^="tag"] {
+    padding-top: 20px !important;
+}
+
+#redoc-container [data-section-id*="/paths/"] {
+    padding-bottom: 20px !important;
+}
+
+#redoc-container h5 {
+    box-sizing: border-box;
+    border: 1px solid #D5D9E0;
+    background-color: #FFFFFF;
+    padding: 16PX;
+}
+
+#redoc-container .dtUibw, #redoc-container .fLUKgj {
```

Comment:
I believe that those "dtUibw" etc. aren't just a random hashes that will change every redoc version, right? :D